system/firewalld.py

```diff
@@ -66,14 +66,19 @@
   state:
     description:
       - "Should this port accept(enabled) or reject(disabled) connections."
-    required: true
+    required: false
```

Comment:
Well, parameter state does not make sense when changing default zone. That was the reason why I made it optional. Does it answer your question? How do you propose to document it?